README.md

```diff
@@ -2,133 +2,140 @@
-* Tokenizer?
-
-# Building
-
-## Windows
-
-* Copy onnxruntime library into the ort/ folder
-  * Can either build Onnxruntime from source in release mode, then copy the files specified in install_ort.bat
-  * Or download a release from https://github.com/microsoft/onnxruntime/releases
-  * Files in ort\lib\ should be:
-    * onnxruntime.dll
-    * onnxruntime.lib
-    * onnxruntime_providers_shared.dll (if using cuda)
-    * onnxruntime_providers_cuda.dll (if using cuda)
-  * Files in ort\include\ should be:
-    * onnxruntime_c_api.h
-* Run the build.bat script to generate build files
-* Open build\Generators.sln in visual studio
-
-To run the python scripts, use PYTHONPATH: `set PYTHONPATH=/path/to/onnxruntime-genai/build/Release/`
-
-## Linux
-
-* Copy onnxruntime library into the ort/ folder
-  * Can either build Onnxruntime from source in release mode, then copy the files specified in install_ort.sh
-  * Or download a release from https://github.com/microsoft/onnxruntime/releases
-  * Files in ort/lib/ should be:
-    * libonnxruntime.so
-    * libonnxruntime.so.(version #)
-    * libonnxruntime_providers_shared.so (if using cuda)
-    * libonnxruntime_providers_cuda.so (if using cuda)
-  * Files in ort/include/ should be
-    * onnxruntime_c_api.h
-* Run the build.sh script to build
-
-To run the python scripts, use PYTHONPATH: `export PYTHONPATH=/path/to/onnxruntime-genai/build/`
-
-## Prerequites
-
-* Onnxruntime
-* cmake
+## Sample code for phi-2 in Python
+
+Install onnxruntime-genai:
+
+```bash
+pip install onnxruntime-genai
+```
+
+```python
+import onnxruntime_genai as og
+
+model=og.Model(f'models/microsoft/phi-2', device_type)
+
+tokenizer = model.CreateTokenizer()
+
+prompt = '''def print_prime(n):
+    """
+    Print all primes between 1 and n
+    """'''
+
+tokens = tokenizer.encode(prompt)
+
+params=og.SearchParams(model)
+params.max_length = 200
+params.input_ids = tokens
+
+output_tokens=model.Generate(params)
+
+text = tokenizer.decode(output_tokens)
+
+print("Output:")
+print(text)
+```
+
+
+## Features
+
+* Supported model architectures:
+  * Phi-2
+  * Llama
+  * GPT
+* Supported targets:   
+  * CPU
+  * GPU (CUDA)
+* Supported sampling features
+  * Beam search
+  * Greedy search
+* APIs
+  * Python
+  * C/C++  
+
+## Coming very soon
+
+* Support for the Mistrel and Whisper model architectures
+* C# API
+* Support for DirectML
+
+## Roadmap
+
+* Automatic model download and cache
+* More model architectures
+
+## Build from source
+
+This step requires `cmake` to be installed.
+
+1. Clone this repo
+
+   ```bash
+   git clone https://github.com/microsoft/onnxruntime-genai
+   ```
+
+2. Install ONNX Runtime
+
+   ```bash
+   mkdir -p ort/include
+   cd ort/include
+   wget https://raw.githubusercontent.com/microsoft/onnxruntime/v1.17.0/include/onnxruntime/core/session/onnxruntime_c_api.h
+   wget https://raw.githubusercontent.com/microsoft/onnxruntime/v1.17.0/include/onnxruntime/core/session/onnxruntime_cxx_api.h
```

Comment:
the onnxruntime_cxx_api.h and onnxruntime_cxx_inline.h files aren't needed, only onnxruntime_c_api.h